app/charts/shared/interaction/tooltip-box.tsx

```diff
@@ -158,9 +175,9 @@ export const TooltipBox = ({
                 height: 0,
                 borderStyle: "solid",
                 top: triangle.top,
-                right: triangle.right,
                 bottom: triangle.bottom,
-                left: triangle.left,
+                left: isMobile ? triangleX : triangle.left,
+                right: !isMobile ? triangle.right : undefined,
```

Comment:
Nit: Would it make sense to move this logic to `mkTriangle` (could be changed to a `useTriangle` hook)? That's a nit, but I think it's not really clear why we have both `triangle.right` and `triangleX` and what is the difference between them. What do you think @noahonyejese? 🤔

Reply:
Yeah I felt the same way about this however moving it to `mkTriangle` would force me to change the `TooltipPlacement` type to accept a new unintuitive type that kinda makes the switch case in `mkTriangle` work and I'd have to change it on a chart level rather than on a tooltip level. 

What do you think @bprusinowski ?

Reply:
Maybe moving it to `mkTriangle` is not the best idea, but then I'd try to find a better name for these variables, or reorganize the code a bit (to clearly see a difference between e.g. `triangle.right` or `.left` and `triangleX`) 🤔

Reply:
Yeah that's a good idea

Reply:
Yes, I think it's much better 🎉 The only thing would be maybe `dynamicTriangleXPosition` -> `mobileTriangleXPosition`? I think that technically, the desktop one (static) is also dynamic, as it's adjusts to where the mouse is, right? If so, maybe the desktop / mobile naming convention would make sense 👍

Reply:
Yep good idea 👍🏽